armi/reactor/components/__init__.py

```diff
@@ -158,7 +158,7 @@ def getComponentArea(self, cold=False):
         Parameters
         ----------
         cold : bool, optional
-            Ignored for this component
+            Compute the area with as-input dimensions, instead of thermally-expanded?
```

Comment:
Why the question mark?

Reply:
Oh, do you not like that?  I usually think of booleans as answering a true or false question.

Reply:
Ah, I can read it that way. When I first read it, in my head it just sounded like it was an unsure statement.

I feel like most docstrings in ARMI are not written in that way, but I'm not necessarily anti.

Reply:
Done